packages/compiler-cli/src/ngtsc/typecheck/extended/src/template_checks/invalid_banana_in_box/index.ts

```diff
@@ -0,0 +1,65 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {TmplAstBoundEvent, TmplAstNode, TmplAstRecursiveVisitor} from '@angular/compiler';
+import * as ts from 'typescript';
+import {ErrorCode} from '../../../../../diagnostics';
+import {TemplateCheck, TemplateContext, TemplateDiagnostic} from '../../../api/api';
```

Comment:
Same here as before - we should always import from the entrypoint:

```suggestion
import {TemplateCheck, TemplateContext, TemplateDiagnostic} from '../../../api';
```

Reply:
This is fixed in #43107